roles/mesos-master/files/haproxy.ctmpl

```diff
@@ -43,6 +43,18 @@ frontend http-in
         # Redirect http to https
         acl is_http hdr(X-Forwarded-Proto) http
         redirect scheme https code 301 if is_http
+
+        {{- $redirect_301 := (print "redirects/" $aws_env "/301") -}}
+        {{- $redirect_302 := (print "redirects/" $aws_env "/302") }}
+        {{ range ls $redirect_301 }}
+        acl {{ .Key }} hdr(host) -i {{ .Key }}
```

Comment:
I would use something like ``redirect-{{ .Key }}`` to avoid any collision in various keys.